frontend/packages/shared/src/components/InputActionWrapper/InputActionWrapper.module.css

```diff
@@ -4,3 +4,20 @@
   align-items: center;
   min-height: var(--fds-component-mode-height-medium);
 }
+
+.standByContainer {
+  position: relative;
+}
+
+.buttonWrapper {
+  display: flex;
+  gap: 2px;
+  background-color: var(--fds-semantic-background-subtle);
+  padding: 1px;
```

Comment:
Should we use rem instead of pixels for the responsiveness?

Reply:
Those pixels should not be there actually. Removed them now 😄